_posts/2023-05-05-NASA-grant-announcement.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: "One small step for NASA, one big step for open research"
+subtitle: "OLS and MetaDocencia to deliver OpenCore Science Training"
+authors:
+- yochannah
+image: https://images.unsplash.com/photo-1502101872923-d48509bff386?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1032&q=80
+photos:
+  name: Photo by Jukan Tateisi on Unsplash
+  url: https://unsplash.com/@tateisimikito
```

Comment:
It should be the link to the image, not the author

Reply:
Done. Thank you!